src/test/java/com/arpnetworking/logback/serialization/steno/StenoSerializationHelperTest.java

```diff
@@ -32,19 +35,30 @@
  */
 public class StenoSerializationHelperTest {
 
-    @Before
-    public void setUp() {
-        _objectMapper = new ObjectMapper();
-        _encoder = new StenoEncoder();
-        _objectMapper.setAnnotationIntrospector(new StenoAnnotationIntrospector(_objectMapper));
+    @Test
+    public void testNonThrowableProxy() throws IOException {
+        final ObjectMapper objectMapper = new ObjectMapper();
+        final StringWriter jsonWriter = new StringWriter();
+        final JsonGenerator jsonGenerator = objectMapper.getFactory().createGenerator(jsonWriter);
+        jsonGenerator.writeStartObject();
+        objectMapper.setAnnotationIntrospector(new StenoAnnotationIntrospector(objectMapper));
+        StenoSerializationHelper.serializeThrowable(
+                new AlternateIThrowableProxy(),
+                jsonGenerator,
+                objectMapper);
+        jsonGenerator.writeEndObject();
+        jsonGenerator.close();
+        Assert.assertEquals(
```

Comment:
Would be nice to see the fields as non-null here to show that it reads the fields from the proxy.